src/Effects/Tremolo.js

```diff
@@ -0,0 +1,83 @@
+Pizzicato.Effects.Tremolo = function(options) {
+
+	this.options = {};
+	options = options || this.options;
+
+	var defaults = {
```

Comment:
Yes, I'll have a look at both of those features if you don't get there first. I think I added wave shape at one point but it was unimpressive, but I might be getting mixed up with another effect. Depth of the wave could be interesting. Stereo I'd also like to add, but perhaps that should be its own effect.